src/e2e.js

```diff
@@ -1,6 +1,17 @@
 /*globals freedom, console, e2e, exports, ArrayBuffer, Uint8Array, Uint16Array, DataView*/
 /*jslint indent:2*/
 
+var useWebCrypto = false;
+if (navigator && navigator.userAgent &&
```

Comment:
Looks like node still considers `navigator` truthy even though it's undefined, at least locally for me, leading to a reference error in the `jasmine_nodejs` task. Switching this to `if (typeof navigator !== 'undefined' && ...` fixes it for me.